sencha-workspace/pages/src/page/TeacherCompetencyDashboard.js

```diff
@@ -29,21 +30,24 @@ Ext.define('Site.page.TeacherCompetencyDashboard', {
         // initialize QuickTips
         Ext.QuickTips.init();
 
-
         // empty content editor container
         teacherDashboardCt.empty();
 
-
         // render teacher dashboard component
         me.dashboard = dashboard = Ext.create('Slate.cbl.view.teacher.Dashboard', {
             renderTo: teacherDashboardCt,
             contentArea: contentAreaData
         });
 
-
         // load data embedded in page
-        Ext.getStore('cbl-students-loaded').loadRawData(studentsData);
+        studentsDataStore = Ext.getStore('cbl-students-loaded');
+
+        studentsDataStore.sorters.add(new Ext.util.Sorter({
```

Comment:
You can just pass the config rather than instantiating the Ext.util.Sorter class here, so delete `new Ext.util.Sorter(` and just call `.add({...})`

See http://docs.sencha.com/extjs/5.0/5.0.1-apidocs/#!/api/Ext.util.Collection-cfg-sorters